rts/motoko-rts-macros/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "motoko-rts-macros"
+version = "0.1.0"
+authors = ["Ömer Sinan Ağacan <omeragacan@gmail.com>"]
```

Comment:
I wonder if we should be avoiding emails since they asked us to purge them. I don't actually care ;->

Reply:
Good point, this should be `dfinity <sdk@dfinity.org>`. I'll update.